mdoc/src/main/scala-3/mdoc/internal/markdown/MarkdownCompiler.scala

```diff
@@ -166,7 +166,7 @@ class MarkdownCompiler(
         case (Right(start), Right(end)) =>
```

Comment:
`pos: SourcePosition` <- will never be optional

Reply:
(comment not related to this line but parameter of this method)

Reply:
I have not noticed! Thanks and fixed!